### PR TITLE
Optimized event init agent AI

### PR DESCRIPTION
**Pull request by AidenTag**
moved gen config to instance vars, removed redundant prompting, set up optimized prompt/exposed datetime data to model, optimized response flow, correctly parses response for output now, added comments for talking points

### CODE REVIEW

File: API/events_initializer.py [hunk 1]
```diff
@@ -95,10 +72,18 @@ async def addEventAIServer(self, action):
...
+                Right now it is {datetime.datetime.utcnow().isoformat() + "Z"} in {pytz.timezone(pytz.country_timezones('US')[0])}
+                """
+
         # Generate response
-        response = chat_session.send_message(prompt + action)
-        print("Response: ", response.text)
+        response = chat_session.send_message(prompt)
         return response
+    
+    # SHOULD WE REALLY BE INSTANTIATING A NEW CHAT THREAD ON EVERY METHOD CALL??? ALSO HOW ARE WE DOING USER THREAD MANAGEMENT???
```
Comment (liamjdavis) [inline API/events_initializer.py:84]: Yes, we should have a separate model initializer. We'll get to work on that next.

**Review by liamjdavis — COMMENTED** [review]
Great work Aiden! We will discuss further architecture next!

File: API/events_initializer.py (hunk 1 above)
Comment (DhyeyMavani2003) [reply API/events_initializer.py:84]: Let's work on that in this same PR! @liamjdavis @AidenTag

**Review by DhyeyMavani2003 — APPROVED** [review]
LGTM!

**Pull request merged**